src/sentry/tagstore/snuba/backend.py

```diff
@@ -726,7 +727,28 @@ def get_group_tag_keys_and_top_values(
             tenant_ids=tenant_ids,
         )
 
-        # Then supplement the key objects with the top values for each.
+        if include_empty_values:
+            keys_to_check = list(values_by_key.keys()) or [k.key for k in keys_with_counts]
+            empty_stats_map = self.__get_empty_value_stats_map(
+                dataset=dataset,
+                filters=filters,
+                conditions=conditions,
+                keys_to_check=keys_to_check,
+                tenant_ids=tenant_ids,
+                start=kwargs.get("start"),
+                end=kwargs.get("end"),
+            )
+            for k, stats in empty_stats_map.items():
+                if not stats or stats.get("count", 0) <= 0:
+                    continue
+                vals = values_by_key.get(k, {})
```

Comment:
nit: from my understanding think these need to be `.get()`, since `empty_stats_map` is either {} (so the iteration won't run) or are guarenteed to have `k` (same with above for `count`)?